selfdrive/car/chrysler/values.py

```diff
@@ -353,6 +356,12 @@ class ChryslerCarInfo(CarInfo):
       b'M2370131MB',
       b'M2421132MB',
     ],
+    (Ecu.gateway, 0x18DACBF1, None): [
+      b'68450806AB',
+      b'68488419AB',
+      b'68535476AB',
+      b'68535476AC',
+    ],
```

Comment:
@realfast do you know why we didn't add gateway before? Do not all trucks have it?

Reply:
We have them, but they are commented out on the 1500s. I am not sure why. It hasn't impacted us.

Reply:
> @realfast do you know why we didn't add gateway before? Do not all trucks have it?
@sshane because it wasn't needed at all to identify the truck. I will work with @vincentw56 to figure out the issue.